core/src/test/java/com/yandex/yoctodb/util/mutable/stored/ManyUniqueValuesTest.java

```diff
@@ -0,0 +1,75 @@
+package com.yandex.yoctodb.util.mutable.stored;
+
+import com.yandex.yoctodb.DatabaseFormat;
+import com.yandex.yoctodb.immutable.Database;
+import com.yandex.yoctodb.mutable.DatabaseBuilder;
+import com.yandex.yoctodb.mutable.DocumentBuilder;
+import com.yandex.yoctodb.util.UnsignedByteArrays;
+import com.yandex.yoctodb.util.buf.Buffer;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static com.yandex.yoctodb.mutable.DocumentBuilder.IndexOption.STORED;
+import static org.junit.Assert.assertEquals;
+
+public class ManyUniqueValuesTest {
+    private String fieldName = "test";
+    @Test
+    public void values100() throws IOException {
+        testDB(100);
+    }
+
+    @Test
+    public void values300() throws IOException {
+        testDB(300);
+    }
+
+    @Test
+    public void values70000() throws IOException {
+        testDB(70000);
```

Comment:
It takes almost 10 seconds that seems to be too much.
Any unit test should either be exhaustive or fast. This one doesn't seem to be exhaustive.

Reply:
But I don't know the other way to test if FoldedArray working correctly with amount of values, there it's more then Short.MAX_VALUE number

Reply:
Maybe we don't need a test for "ManyUniqueValues", but we need it for ManyNonUniqueValuesTest
The test build the database using the folding index for a huge number of non-uniques values, then read it and assert if there correct values inside